ci/build_darwin.sh

```diff
@@ -0,0 +1,44 @@
+#!/bin/bash -e
+#
+# Build Snowflake Python Connector on Mac
+# NOTES:
+#   - To compile only a specific version(s) pass in versions like: `./build_darwin.sh "3.5 3.6"`
+PYTHON_VERSIONS="${1:-3.5 3.6 3.7 3.8}"
+THIS_DIR="$(cd "$(dirname "${BASH_SOURCE[0]}")" && pwd)"
+CONNECTOR_DIR="$(dirname "${THIS_DIR}")"
+DIST_DIR="$CONNECTOR_DIR/dist"
+
+cd $CONNECTOR_DIR
+# Clean up previously built DIST_DIR
+if [ -d "${DIST_DIR}" ]; then
+    echo "[WARN] ${DIST_DIR} already existing, deleting it..."
+    rm -rf "${DIST_DIR}"
+fi
+mkdir -p ${DIST_DIR}
+
+# Make sure we build for our lowest target
+# Should be kept in sync with .github/worklfows/build_test.yml
+export MACOSX_DEPLOYMENT_TARGET="10.12"
```

Comment:
Let's change this AFTER you merge this change and the next release is out, then bump up the minor version.

Reply:
Just updated it

Reply:
Should we remove it? I could bump minor version in this PR

Reply:
Oh after the next release, okay then I'll revert this and file a Jira against myself

Reply:
yes, let's synchronize this change with the Python connector version bump. The test environment OSX version change is fine. We already have been doing it on 10.13 since a couple of months ago.